### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Isabella-Veloso**
Adding API removal process to README.

#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [x] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,6 +293,29 @@ To update the left navigation after changing an endpoint's path follow the steps
 4. Send the PR in `#dev-portal-pr` Slack channel for approval.
 5. After approval, merge the PR.
 
+### How to remove an API reference from Developer Portal?
+
+You will need to create two pull requests in the following order: one for `devportal` repository and other for `openapi-schemas`.
+
+To remove the API content from `devportal` repository, follow the steps below:
```
Comment (julia-rabello) [inline README.md:300]:
```suggestion
To remove the API content from `devportal`, follow the steps below:
```

File: README.md [hunk 2]
```diff
@@ -293,6 +293,29 @@ To update the left navigation after changing an endpoint's path follow the steps
 4. Send the PR in `#dev-portal-pr` Slack channel for approval.
 5. After approval, merge the PR.
 
+### How to remove an API reference from Developer Portal?
+
+You will need to create two pull requests in the following order: one for `devportal` repository and other for `openapi-schemas`.
```
Comment (julia-rabello) [inline README.md:298]:
```suggestion
You will need to create two pull requests, the first one in [devportal](https://github.com/vtexdocs/devportal), and the second one in [openapi-schemas](https://github.com/vtex/openapi-schemas). Changes are detailed below.
```

File: README.md [hunk 3]
```diff
@@ -293,6 +293,29 @@ To update the left navigation after changing an endpoint's path follow the steps
...
+To remove the API content from `devportal` repository, follow the steps below:
+
+1. Open a branch in the [devportal](https://github.com/vtexdocs/devportal) repository.
+2. In the `src/pages/api/openapi/[slug].tsx` file, remove the content that refers to the API. It follows this syntax: `'{schemaFileName}': '{slug}'`.
+3. In the `src/utils/getReferencePaths.ts` file, remove the content that refers to the API. It follows this syntax: `'{schemaFileName}': '{slug}'`.
+4. Open a PR with the changes.
+5. Send the PR in `#dev-portal-pr` Slack channel for approval.
+6. After approval, merge the PR.
+
+Next, to remove the API content from [openapi-schemas](https://github.com/vtex/openapi-schemas) repository, follow the steps below:
```
Comment (julia-rabello) [inline README.md:309]:
```suggestion
Next, to remove the API content from `openapi-schemas`, follow the steps below:
```

File: README.md [hunk 4]
```diff
@@ -293,6 +293,29 @@ To update the left navigation after changing an endpoint's path follow the steps
...
+
+1. Open a branch in the `openapi-schemas` repository.
```
Comment (julia-rabello) [inline README.md:311]:
```suggestion
1. Open a branch in the [openapi-schemas](https://github.com/vtex/openapi-schemas) repository.
```

File: README.md [hunk 5]
```diff
@@ -293,6 +293,29 @@ To update the left navigation after changing an endpoint's path follow the steps
...
+### How to remove an API reference from Developer Portal?
+
+You will need to create two pull requests in the following order: one for `devportal` repository and other for `openapi-schemas`.
+
+To remove the API content from `devportal` repository, follow the steps below:
+
+1. Open a branch in the [devportal](https://github.com/vtexdocs/devportal) repository.
+2. In the `src/pages/api/openapi/[slug].tsx` file, remove the content that refers to the API. It follows this syntax: `'{schemaFileName}': '{slug}'`.
+3. In the `src/utils/getReferencePaths.ts` file, remove the content that refers to the API. It follows this syntax: `'{schemaFileName}': '{slug}'`.
+4. Open a PR with the changes.
```
Comment (julia-rabello) [inline README.md:305]: I suggest adding a step to remove the API and its endpoints from navigation.json

Comment (julia-rabello) [reply README.md:305]: Maybe it's worth adding the redirect as an optional step here instead of in a callout in the end, since it's in the same repository

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Isabella-Veloso) [reply README.md:305]: Agree and changed it!

**Review by julia-rabello — APPROVED** [review]

**Pull request merged**